pkg/apis/batch/validation/validation.go

```diff
@@ -162,6 +162,13 @@ func ValidateCronJob(scheduledJob *batch.CronJob) field.ErrorList {
 	// CronJobs and rcs have the same name validation
 	allErrs := apivalidation.ValidateObjectMeta(&scheduledJob.ObjectMeta, true, apivalidation.ValidateReplicationControllerName, field.NewPath("metadata"))
 	allErrs = append(allErrs, ValidateCronJobSpec(&scheduledJob.Spec, field.NewPath("spec"))...)
+	if len(scheduledJob.ObjectMeta.Name) > 52 {
```

Comment:
Replace that maybe with `DNS1035LabelMaxLength-11` from `k8s.io/apimachinery/pkg/util/validation` package.